health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/attendance_inbox/filter_section.js

```
@@ -0,0 +1,136 @@
+        >
+          <div style={{ display: "flex", alignItems: "center" }}>
+            <SVG.FilterAlt width={"32px"} height={"32px"} fill={"#c84c0e"} />
+            <span className="custom-inbox-filter-heading">{t("HCM_AM_FILTER")}</span>
+          </div>
+
+          <span
+            onClick={() => {
+              setReset(true);
+            }}
+            style={{ border: "1px solid #e0e0e0", padding: "6px", marginBottom: "10px", cursor: "pointer" }}
+          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)